patch/0001-armhf-default-config-for-the-sonic-patches.patch

```diff
@@ -0,0 +1,82 @@
+From 3a09f684413faf1c4f7d21d7d79914b54e087cf6 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 10:48:29 +0530
+Subject: [PATCH] armhf default config for the sonic patches
```

Comment:
why do you need this file?

Reply:
Due to patches in Kconfig, kernel make enters into into interactive mode for user to provide (N/m/y) options. 
Few configs like Mellanox Platform are not relevant but still config has to set with "N" (not set) option.
To avoid all manual changes in config, I have generated "default" configs for ARMHF and ARM64 after applying patch with "not set/N" option, wich reduced manual effort of finding right place to insert CONFIG variables.